README.md

```diff
@@ -0,0 +1,18 @@
+# rclcpp
+
+This repository contains the source code for the ROS Client Library for C++ package, included with a standard install of any ROS 2 distro.
+
+rclcpp provides the accepted, standard C++ API for interacting with ROS 2.
+
+## Usage
+
+`#include "rclcpp/rclcpp.hpp"` allows use of the most common elements of the ROS 2 system.
+
+Visit the [rclcpp API documentation](http://docs.ros2.org/dashing/api/rclcpp/) for a complete list of its main components.
```

Comment:
could that maybe point to eloquent at this point? I don't know if docs.ros2.org has a substitution for `latest` or so to keep it also pointing to the latest version of the API doc.

Reply:
I got to the API docs from ros index, which doesn't seem to have eloquent as a distro option on package search result pages. That seems like a problem...